packages/components/src/components/buttons/Pin/index.native.tsx

```diff
@@ -1,12 +1,35 @@
+import React from 'react';
 import styled from 'styled-components/native';
 import colors from '../../../config/colors';
 
-const ButtonPin = styled.TouchableHighlight`
+const Touchable = styled.TouchableHighlight`
     width: 80px;
     height: 80px;
     border: 1px solid ${colors.DIVIDER};
     background: ${colors.WHITE};
     position: relative;
 `;
 
+const Dot = styled.View`
+    width: 6px;
+    height: 6px;
+    position: absolute;
+    border-radius: 6px;
+    background: ${colors.TEXT_PRIMARY};
+    top: 37px;
+    left: 37px;
```

Comment:
:D isn't there a more flexible way to center the element horizontally? at least calc() just like in web?

Reply:
Yes ideal would be calc(50% - 3px) but react native doesn't have calc() yet but it's in progress: https://github.com/kristerkari/react-native-css-modules/issues/9

Reply:
tried something in https://github.com/trezor/trezor-suite/pull/272/commits/c9441f1690bf7836d1873e74613be328533413d0 😬